CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ All notable changes to this project are documented in this file.
  - Fixed `LineEdit`'s placeholder text not being rendered with the same font attributes as regular `LineEdit` text.
  - Fixed SVG with text (#2646)
  - Software renderer: Show the cursor in TextInput
+ - Add a type `component-factory` to the Slint language, taking a factory object used to create Components
```

Comment:
Any changelog item that says "Added ... to the Slint Language" belongs perhaps into the "Slint Language" section :-)

If it were there, I'd call it

```
 - Added a `component-factory` property type that can dynamically create components at run-time.
```